src/Command/CommandProvider.php

```diff
@@ -37,7 +37,7 @@ private static function siteCommands(Container $container, Application $console)
 
     private static function pharCommands(Container $container, Application $console): void
     {
-        if (\Phar::running() === '') {
+        if (!($_SERVER['INCLUDE_SELF_UPDATE'] ?? false) && \Phar::running() === '') {
```

Comment:
Why this change? Where INCLUDE_SELF_UPDATE is defined?

Reply:
This makes it possible to run `INCLUDE_SELF_UPDATE=1 ./bin/console` in order to include that command and test the help output without building a phar

Reply:
If you don't think it's worth including I'm happy to remove it. It's easy enough to comment out the early return in dev if needed.

Reply:
What about `if (!empty($_ENV['CONCRETE_INCLUDE_PHAR']) ... ` ?

Reply:
Sorry, without the `!`

Reply:
The reason I went with _SERVER over _ENV is because my local environment has variables_order set to not populate _env which is pretty common

Reply:
Let's just drop it for now and add it back later if there's value